cdap-api-spark/src/main/scala/co/cask/cdap/api/spark/SparkMain.scala

```diff
@@ -162,12 +201,12 @@ trait SparkMain extends Serializable {
     /**
       * Creates a [[org.apache.spark.rdd.RDD]] from the given [[co.cask.cdap.api.dataset.Dataset]].
       *
-      * @param datasetName name of the Dataset
-      * @param arguments arguments for the Dataset; default is an empty [[scala.collection.Map]]
+      * @param datasetName name of the dataset
```

Comment:
@chtyim I thought there was a documentation task to reduce this upper-casing. I think if we just upper-case it without an ``{@link Dataset}``, it looks a little strange.